geopandas/tests/test_geodataframe.py

```diff
@@ -466,6 +466,12 @@ def test_coord_slice_points(self):
         assert_frame_equal(self.df2.loc[5:], self.df2.cx[:, 5:])
         assert_frame_equal(self.df2.loc[5:], self.df2.cx[5:, 5:])
 
+    def test_from_dict(self):
+        data = {"A": [1], "geometry": [Point(0.0, 0.0)]}
+        df = GeoDataFrame.from_dict(data, crs=3857)
+        assert df.crs == "epsg:3857"
+        assert df._geometry_column_name == "geometry"
```

Comment:
Can you also add a test with a different geometry column name?

Reply:
Will do